data-pipeline/bcreg/bcregistries.py

```diff
@@ -735,6 +735,26 @@ def get_max_event_date(self):
             if cur is not None:
                 cur.close()
 
+    # get the last event date on or before now
+    def get_max_date_before_now(self):
+        cur = None
+        try:
+            # create a cursor
+            cur = self.conn.cursor()
+            cur.execute("""SELECT max(event_timestmp) FROM """ + BC_REGISTRIES_TABLE_PREFIX + """event where event_timestmp <= NOW()""")
```

Comment:
`NOW()` is system clock dependent, likely UTC in most cases.  Where the entries in the event tables are assuming PDT/PST.

Reply:
I tested out this sql specifically and it seems to be working as intended

Reply:
Yeah it will depend on what timezone is set in Postgres.

Wondering if we should add a line to specifically set the timezone to 'PST' before the scripts execute the SQL commands?

https://www.postgresql.org/docs/13/sql-set.html

Reply:
Straight from the docs `SET TIME ZONE 'PST8PDT';` we can set the runtime parameters.